Abhi-AnimeX/app/src/main/java/com/abhijith/animex/domain/mapper/AnimeMapper.kt

```diff
@@ -0,0 +1,22 @@
+package com.abhijith.animex.domain.mapper
+
+import com.abhijith.animex.data.model.AnimeEntity
+import com.abhijith.animex.domain.model.AnimeItem
+
+object AnimeMapper {
```

Comment:
why object?

Reply:
Removed the entire object and created an extension function instead in the AnimeEntity Class. @anandwana001